concepts/05 Widgets/FileUploader/99 How To/10 Specify a Header to the Request.md

```diff
@@ -0,0 +1,113 @@
+Use the [uploadHeaders](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#uploadHeaders) option to specify a header to the request.
```

Comment:
How can you "specify a header to the request"? What it mean when you 'specify' a header to a request?

Reply:
[Request headers](https://developer.mozilla.org/en-US/docs/Glossary/Request_header) contain additional information passed with a request.

Request header has the following format: {header_name) : {header value}

For example request headers for a request:

Host: developer.mozilla.org
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:50.0) Gecko/20100101 Firefox/50.0
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-US,en;q=0.5 
// Added a new header
my-own-header: test-value

"Specify a header for a request" is an usual phrase in this context.

Reply:
Specify 'for', but not 'to'